Clinical-Genomics/cgweb
# hides qc text for external

This PR hides the qc-button text for external samples

How to test:
1. install
1. create external order
1. add family

Expected outcome:
No QC-button text visible when editing family
